bin/cleanup_old_instances.rb

```diff
@@ -1,4 +1,8 @@
-old_instances = Kuroko2::JobInstance.where('finished_at < ?', 3.months.ago)
+target_date = 3.months.ago
+old_instances = Kuroko2::JobInstance
+  .where('finished_at < ?', target_date)
+  .or(Kuroko2::JobInstance.where('canceled_at < ?', target_date))
+  .or(Kuroko2::JobInstance.where('error_at < ?', target_date))
```

Comment:
`ERROR` status is not a final state.
https://github.com/cookpad/kuroko2/blob/6a777d0950141324f4799fc08388af3211ce264d/docs/user_guide.md#job-status-flow

I want you to delete this condition.

Reply:
Oops... Rebased